includes/definitions/discovery/ionodes.yaml

```diff
@@ -0,0 +1,35 @@
+mib: IF-MIB:IONODES-IONSERIES-MIB
```

Comment:
You do not use IF-MIB, do you ? if not, you can remove it from here and only keep IONODES-IONSERIES-MIB